modules/kubernetes-addons/aws-coredns/README.md

```diff
@@ -22,9 +22,18 @@ To enable and modify the EKS managed addon for CoreDNS, you can reference the fo
 
 ```sh
 kubectl --namespace kube-system delete deployment coredns
+
 kubectl --namespace kube-system annotate --overwrite service kube-dns meta.helm.sh/release-name=coredns
 kubectl --namespace kube-system annotate --overwrite service kube-dns meta.helm.sh/release-namespace=kube-system
 kubectl --namespace kube-system label --overwrite service kube-dns app.kubernetes.io/managed-by=Helm
+
+kubectl --namespace kube-system annotate --overwrite configmap coredns meta.helm.sh/release-name=coredns
+kubectl --namespace kube-system annotate --overwrite configmap coredns meta.helm.sh/release-namespace=kube-system
+kubectl --namespace kube-system label --overwrite configmap coredns app.kubernetes.io/managed-by=Helm eks.amazonaws.com/component=coredns-
+
+kubectl --namespace kube-system annotate --overwrite serviceaccount coredns meta.helm.sh/release-name=coredns
+kubectl --namespace kube-system annotate --overwrite serviceaccount coredns meta.helm.sh/release-namespace=kube-system
+kubectl --namespace kube-system label --overwrite serviceaccount coredns app.kubernetes.io/managed-by=Helm eks.amazonaws.com/component=coredns-
```

Comment:
Perhaps it would be better to avoid the mentioning the removal of `eks.amazonaws.com/component=coredns` here, since it's not strictly necessary.

I'd also be interested to make this more readable by combining `kubectl annotate` calls and inserting some newlines, if y'all are amenable.